types/decimal.go

```diff
@@ -474,6 +474,18 @@ func (d Dec) String() string {
 	return string(bzStr)
 }
 
+func (d Dec) Float64() (float64, error) {
```

Comment:
Godoc missing. May need to come with a warning

Reply:
fixed now